cmd/minishift/cmd/root.go

```diff
@@ -141,6 +140,11 @@ var RootCmd = &cobra.Command{
 		}
 
 		setDefaultActiveProfile()
+
+		// Adding minishift version information to debug logs
+		if glog.V(2) {
+			fmt.Println(fmt.Sprintf("-- minishift version: %s", version.GetMinishiftVersion()))
```

Comment:
Should be `"-- Minishift version: %s"` as `minishift` (small letter) indicates command.